lerna.json

```diff
@@ -1,8 +1,4 @@
 {
-  "packages": [
-    "packages/*",
```

Comment:
I believe both are needed

Reply:
I don't think so.

```
if --use-workspaces is true then packages will be overridden by the value from package.json/workspaces.
```

According to https://github.com/lerna/lerna/blob/master/commands/bootstrap/README.md#--use-workspaces

And we can see in the build it builds all 9 packages

Reply:
Okay!